common/src/main/java/io/github/gaming32/bingo/game/BingoGame.java

```diff
@@ -745,6 +823,7 @@ public BingoGame createGame(Scoreboard scoreboard) throws IllegalStateException
 
             game.baseStats.putAll(baseStats);
 
+            game.remainingTeams = playingTeams.orElseGet(() -> BingoBoard.Teams.fromAll(teams.length));
```

Comment:
This gives all teams only two minutes to join back after the server starts. This makes me think in general that the game's auto resign timer should be frozen if no Bingo players are online.

Reply:
Yeah sure, I can add a check for if the server is empty. I didn't think about that situation

Reply:
I did it slightly differently. The auto-resign won't trigger if nobody from that team has been online since server restart